cmd/tempo/app/app.go

```diff
@@ -410,7 +410,7 @@ func (t *App) statusHandler() http.HandlerFunc {
 					if err == nil {
 						err = e
 					} else {
-						err = errors.Wrap(err, e.Error())
+						err = fmt.Errorf(e.Error(), ": %w", err)
```

Comment:
I think should be like this:
```suggestion
						err = fmt.Errorf("%s: %w", e.Error(), err)
```
Maybe we could also use `errors.Join()` or dskit multierror instead of `joinErrors`?

Reply:
AFAIK about how `errors.Join()` would parse the error message, it uses a newline separator. I'm under the assumption we're looking to have single-line rather than multi-line errors.

E.g.
```
error writing status endpoints: <err>

vs.

<err_one>
<err_two>
```

Curious if `dskit` would format it similar to the latter?

Reply:
Your are right, using dskit multi error or errors.Join will both alter the format or the error string.
Regarding dskit: for 5 errors the multierror creates a string that looks like this:
```
5 errors: err1; err2; err3; err4; err5
```
I think it should be fine to use dskit multierror here. But it is also completely OK to keep the `joinErrors`.

Reply:
Maybe we leave it as-is for now, for consistency's sake? I'd be happy to revisit it again in the future, if need be.